src/components/FeedbackForm/FeedbackForm.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useContext, useState } from "react"
+
+import { FeedbackContext } from "../../context/FeedbackContext"
+import type { FeedbackMetadataAndComment } from "../../types/feedbackTypes"
+import { BASE_URL } from "../../config/constants"
+
+/**
+ * Component that wraps the DS Feedback box. Can be opened by clicking the button rendered
+ * or by clicking on 'Contact a librarian' (located in src/app/components/Item/InformationLinks.jsx)
+ */
+const FeedbackForm = () => {
+  const [screen, setScreen] = useState("form")
```

Comment:
can this variable name reflect that its used to populate the view property in the feedback component? It's only used for that, correct?

Reply:
Good call, fixed.